Base/res/ladybird/about-pages/settings.html

```diff
@@ -372,6 +388,13 @@ <h1>Ladybird Settings</h1>
         <div class="card">
             <div class="card-header">Privacy</div>
             <div class="card-body">
+                <div class="card-group inline-container">
+                    <span>Browsing Data</span>
+                    <button id="clear-browsing-data" class="secondary-button">Remove...</button>
```

Comment:
The verb you use in the modal dialog is "Clear" in the title but "Remove" in the primary action. I'm slightly leaning towards using "Clear" everywhere, but I don't mind as long as it's consistent :)

Reply:
Went with "Clear"